src/index.css

```diff
@@ -104,6 +105,65 @@ h1 {
   margin-top: 8px;
 }
 
+nav {
+  background-color: #262626;
+  width: 100%;
+  position: sticky;
+  top: 0;
+}
+
+nav li {
+  font-size: 19px;
+  font-weight: 700;
+  line-height: 26px;
+  text-align: center;
+  margin-bottom: 8px;
+}
+
+nav a:hover,
+a:focus {
+  background-color: #797876;
+}
```

Comment:
The keyboard navigation works flawlessly now 👏 
The menu items get highlighted on focus and are also entirely clickable ✨ 
Really nice! 🙌 

https://github.com/user-attachments/assets/21d30a3b-94ac-40f9-b9db-ee57d212381f